sql/testdata/explain_debug

```diff
@@ -49,6 +53,38 @@ UPDATE abc SET c = NULL WHERE a = 1
 query ITTB
 EXPLAIN (DEBUG) SELECT * FROM abc
 ----
-0  /abc/primary/1/'one'    NULL  true
-1  /abc/primary/2/'two'    NULL  true
-2  /abc/primary/3/'three'  NULL  true
+0  /abc/primary/1/'one'      NULL  true
+1  /abc/primary/2/'two'      NULL  NULL
+1  /abc/primary/2/'two'/c    2.2   true
+2  /abc/primary/3/'three'    NULL  NULL
+2  /abc/primary/3/'three'/c  3.3   true
+
+statement error duplicate key value .* violates unique constraint
+INSERT INTO abc VALUES (4, 'four', 3.3, NULL)
```

Comment:
actually, i'll fold them into the existing test/update/delete tests; i don't think that's inappropriate